Makefile

```diff
@@ -16,20 +16,20 @@ bootstrap-oss:
 	@$(MAKE) xcodeproj-oss
 	@$(MAKE) install-mm
 	@echo "  ✅ Bootstrapped! Opening Xcode..."
-	@sleep 1 && xed .
+	@open -g PointFree.xcworkspace
```

Comment:
The `xed` command defaults to opening as an SPM package. Since we still rely on the project file for playgrounds, let's open it using `open` instead. I use the `-g` flag to open it in the background to avoid interrupting anything we might still wanna do in the terminal, but we could open in the foreground instead if you prefer!